src/services/utilities.ts

```diff
@@ -407,8 +407,11 @@ namespace ts {
         return start < end;
     }
 
+    /**
+     * Assumes `candidate.start <= position` holds.
+     */
     export function positionBelongsToNode(candidate: Node, position: number, sourceFile: SourceFile): boolean {
-        return candidate.end > position || !isCompletedNode(candidate, sourceFile);
+        return position < candidate.end || !isCompletedNode(candidate, sourceFile);

```

Comment:
I can tell you have a preference for ordering comparisons as I've seen you do this in several pull requests (probably based on your math background :wink:), but please don't go changing existing expressions unless you are re-writing that code anyway, they are unreadable as written, or they are incorrect.